### ISSUE TITLE
Green Stripes

### ISSUE THREAD

**Issue by HerrBach2001** [Anuken/Mindustry#3820]
**Platform**: Windows 10 64 Bit

**Build**: Build20247 

**Issue**: The complete game is green or 

with light green stripes.

**Steps to reproduce**:  The bug occurred while loading the game. The game did not crash.

**Link(s) to mod(s) used**: No mods installed or used.
**Save file**: [Game progress.zip](https://github.com/Anuken/Mindustry/files/5648947/Game.progress.zip)

If you remove the line above without reading it properly and understanding what it means, I will reap your soul. Even if you're playing on someone's server, you can still save the game to a slot. **I'm understanding**

**(Crash) logs**: It's not a crash.

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [X] **I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.**
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by Anuken** [comment]
> I have searched the closed and open issues to make sure that this problem has not already been reported.

No, you haven't.

**Comment by HerrBach2001** [comment]
how cab I fix it?

**Comment by Anuken** [comment]
You can't.

**Comment by DeltaNedas** [comment]
update your gpu drivers:
1. install debian
2. `apt install libgl1-mesa-glx`

**Comment by magicc22** [comment]
I have the same issue. I loved v104, worked great. updated today to latest and the graphics have horizontal green stripes and is unreadable.  ( i can make out where the menu is and exit :-) )
Also Windows 10, 64 bit.
I updated my graphics driver for my graphics card: NVIDIA GeForce 940MX to no avail.
I tried going back versions, and the first version with this issue is v106.
Would a screen shot help?
I see this is closed today, is there a solution posted that I am missing? 
Would love to play the new version!
Thanks,
Chris
